specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2019-03-01-preview/savedquery.json

```diff
@@ -0,0 +1,537 @@
+            "schema": {
+              "$ref": "#/definitions/SavedQueryResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The newly created saved query.",
+            "schema": {
+              "$ref": "#/definitions/SavedQueryResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/SavedQueryError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create Saved Query": {
+            "$ref": "./examples/SavedQueryAdd.json"
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates a saved query that has already been added.",
+        "operationId": "SavedQuery_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SavedQueryResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "description": "Properties that need to be specified to create a new saved query.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SavedQueryUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The saved query definition updated.",
+            "schema": {
+              "$ref": "#/definitions/SavedQueryResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/SavedQueryError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update Saved Query": {
+            "$ref": "./examples/SavedQueryUpdate.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ResourceGraph/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all of the available REST API operations.",
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "OperationsList": {
+            "$ref": "./examples/OperationsList.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
```

Comment:
This is redundant since resourcegraph.json already has it.

Reply:
The resorucegraph.json has a lot more properties than it needs so we decided to use a slim version.